examples/with-aws-amplify-typescript/pages/index.tsx

```diff
@@ -140,16 +140,25 @@ const App = (props: Props) => {
     </div>
   )
 }
-App.getInitialProps = async () => {
+
+export const getStaticProps = async () => {
   let result: { data: GetTodoListQuery; errors: {}[] } = await API.graphql(
     graphqlOperation(getTodoList, { id: 'global' })
   )
   if (result.errors) {
     console.log('Failed to fetch todolist. ', result.errors)
-    return { todos: [] }
+    return {
+      props: {
+        todos: [],
+      },
+    }
   }
   if (result.data.getTodoList !== null) {
-    return { todos: result.data.getTodoList.todos.items }
+    return {
+      props: {
+        todos: result.data.getTodoList.todos.items,
+      },
+    }
   }
 
   try {
```

Comment:
Do we need this try/catch? not sure what it does

Reply:
The method is used as some kind of "migration" to initialize the collection @ the database.